centOS packages are not installing correctly in E2E Vulnerability detection for python310

# Description

It was detected in an E2E vulnerability detection tests report (by @QU3B1M) that the centOS packages are not installing correctly in case of running these tests with a 3.10.12, instead of the 3.9 used in the automation pipeline.

Report's logs show these errors:
```
2024-09-04 00:15:59 Error installing package on agent4: expected string or bytes-like object (remote_operations_handler.py:335)
2024-09-04 00:15:59 Error installing package on agent1: expected string or bytes-like object (remote_operations_handler.py:335)
```
Marking the operation_results check as failed
```
E         Check operation_successfull_for_all_agents failed. Evidences (['operation_results']) can be found in the report.
```

# Tasks
- [x] Research installation error using python 3.10.12
- [x] Update the E2E framework to support the installation of CentOS packages across all versions. 

# Validation
- [x] Install with success a centos package using  python 3.10.12


# Evidences
- [report.zip](https://github.com/user-attachments/files/16870145/report.2.zip)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [4.10.0] - TBD
 
+### Fixed
+
+- Fix E2E VD tests not working if the Python version is different to 3.9 ([#5787](https://github.com/wazuh/wazuh-qa/pull/5787)) \- (Framework)
+
 ## [4.9.1] - TBD
 
 ### Added

--- a/deps/wazuh_testing/wazuh_testing/tools/system.py
+++ b/deps/wazuh_testing/wazuh_testing/tools/system.py
@@ -513,8 +513,7 @@ class HostManager:
             return result
 
         def install_yum_package(host, url):
-            result = self.get_host(host).ansible("yum", f"name={url} state=present "
-                                                 'disable_gpg_check=True', check=False)
+            result = self.get_host(host).ansible("command", f"yum install -y {url}", check=False)
             return result
 
         def install_pkg_package(host, url):

